src/pages/RegisterPage.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button, PasswordInput, TextInput } from "@mantine/core";
+import { useState } from "react";
+import { CreateUserDto } from "../types/dto";
+import createUser from "../api/users";
```

Comment:
import { register }  from "../api/auth";

Reply:
told to ignore